src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js

```diff
@@ -231,11 +231,21 @@ const generateAudioBubble = (
       <MessageBubble author={message.authorName}>
         <div>
           <iframe
-            src={src}
-            title="SUSI features audio"
-            frameBorder="0"
-            allowFullScreen
-          />
+            id="sc-widget"
+            title="SUSI Soundcloud"
+            src={`https://w.soundcloud.com/player/?url=${src}`}
+            width="100%"
+            height="20rem"
```

Comment:
This way the height is not applied in rem. Use style to apply height in rem. Also, increase the height than the current one.

Reply:
Fixed it